source/features/sticky-sidebar.css

```diff
@@ -4,6 +4,9 @@
 
 .rgh-sticky-sidebar {
 	position: sticky;
+}
+
+:not(#partial-discussion-sidebar).rgh-sticky-sidebar {
```

Comment:
Is there anything better than :not? Like `.readme .sticky`

Reply:
Not that I've found. We could do `.BorderGrid--spacious.rgh-sticky-sidebar` but that's not much better

Reply:
Is this change needed at all? It looks like it will just be overridden by the next selector